docs/decisions/0011-managing-logging-configuration.rst

```diff
@@ -0,0 +1,67 @@
+Status
+======
+
+Draft
+
+
+Context
+=======
+
+Logging configuration needs more options to be useful for devops/SRE work.
+
+Decision
+========
+
+Let us add an optional override dictionary to `get_logging_config`. This would
```

Comment:
The general idea is to be able to support customizing the logging configuration by passing in configuration values. The specifics of the mechanism are still subject to debate and design work. For the time being we have written a plugin to override the `LOGGING` setting but it might be worth exploring how to support some measure of modification without having to resort to a plugin.